docs/guides/session_management.md

```diff
@@ -0,0 +1,161 @@
+---
+id: sessionmanagement
+title: Session Management
+---
+[`SessionPool`](../api/sessionpool) is a class that allows you to handle the rotation of proxy IP addresses along with cookies and other custom settings in Apify SDK.
```

Comment:
Why is this important, what problem does it solve? Basically, don't assume the readers implicitly have certain knowledge and just repeat it to make sure.